app/models/classification.rb

```diff
@@ -23,6 +23,18 @@ class Classification < ActiveRecord::Base
   scope :created_by, -> (user) { where(user: user) }
   scope :complete, -> { where(completed: true) }
 
+  after_create do
```

Comment:
i'm not a huge fan of callbacks that modify state in associated models, i can see the reasoning here but any reason why we can't extract these to service objects and then just link them via the controller / call them when testing?